confluent-docker-utils.yaml

```diff
@@ -29,16 +30,14 @@ environment:
 pipeline:
   - uses: git-checkout
     with:
-      expected-commit: 4a3cd161f5ee6bdf711396aa06de8121e0e3600f
+      expected-commit: 37fde330729139fda949820ddd12727e76ab7ec6
       repository: https://github.com/confluentinc/confluent-docker-utils
       tag: v${{package.version}}
 
   - runs: |
       # Workaround to fix `AttributeError: cython_sources` issue
       # https://github.com/yaml/pyyaml/issues/724#issuecomment-1638587228
-      sed -i 's|PyYAML~=5.4.1|PyYAML==6.0.1|g' requirements.txt
-      # Fix CVE GHSA-h4gh-qq45-vh27
-      sed -i 's|cryptography~=42.0.4|cryptography~=43.0.1|g' requirements.txt
+      echo 'PyYAML==6.0.1' >> requirements.txt
```

Comment:
The latest tag removed 'PyYAML~=5.4.1' from the requirements.txt. However the original issue this line was fixing still existed, and we still need yYAML==6.0.1. This was changed to write the entry and not try and update it.

I tried smarter things, like this:

```bash
pip install --root=${{targets.destdir}} --prefix=/usr --prefer-binary PyYAML==6.0.1
```

But it didn't work. Note also line 52 in the Melange. TLDR, this fixes it, other attempts to a more refined approach failed.